### ISSUE TITLE
Installed but not opening dialog box

### ISSUE THREAD

**Issue by giociva** [q2g/q2g-con-psexecute-qvx#13]
Hi,
I try to install the PS Connector. Download and extracted .zip
Copied the 2 folders q2gConPSExecuteqVX and q2gHypercubeQVX (copying the whole extension package don't works) into Program Files/Common Files/Qlik/Custom Data.
I can see the connectors when I try to create a new data connection, but clicking on them do not opens any box for configuration. Qlik Sense Feb 2018. Thanks.

**Comment by konne** [comment]
Hi,

it looks like the same issue q2g/q2g-con-hypercube-qvx#20
@mossbeard please test direct after easter write your results.

**Comment by konne** [comment]
Hi,
@giociva we have found the issue with Qlik Feb 2018 and fixed it.
Please donwload the new extension package version.

**Comment by giociva** [comment]
Still having same problem here :(

**Comment by konne** [comment]
I will check

**Comment by konne** [comment]
I did't forgort you, I will come back tomorrow on that problems

**Comment by giociva** [comment]
any update on this? thanks

**Comment by mossbeard** [comment]
Sorry for the late answer.
Has your problem with the connector resolved?
I could not understand the sample with the current version (November 2018) of qlik.

**Comment by giociva** [comment]
Nope, problem still persists. no dialog box. Nov 2018 Patch 2 Release.

**Comment by mossbeard** [comment]
@giociva
Hi, can we do a teamviewer session with you? We are locaded in Germany. 
The best time for us is 3pm - 6pm CET.

An idea that came to my mind.
Have you installed a special browser plugin which blocks the dialog?

**Comment by fireRoads** [comment]
I am having similar problem on Qlik Sense Enterprise April 2019 Server. Using the version that is provided when downloading the zip-file. I can see the connector in the Create Data Connection dialog but when selecting it the whole page turns white and I get the following errors i the Chrome console:

WARNING: Tried to load AngularJS more than once.
5.15a5e6823cb402d6433f.js:8 Uncaught (in promise) Error: [ng:btstrpd] App already bootstrapped with this element 'document'
https://errors.angularjs.org/1.7.2/ng/btstrpd?p0=document
    at 5.15a5e6823cb402d6433f.js:8
    at a (5.15a5e6823cb402d6433f.js:8)
    at Object._e [as bootstrap] (5.15a5e6823cb402d6433f.js:8)
    at Module.a.bootstrap (setup-view.aaafe511db98d706b7b3.js:8)
    at e (<anonymous>:7:46480)
    at d (app.6.43.9.js:7)
    at Generator.o [as _invoke] (app.6.43.9.js:7)
    at Generator.e.<computed> [as next] (app.6.43.9.js:7)
    at f (<anonymous>:7:45642)
    at a (<anonymous>:7:45843)

Therefore I cannot add the data connection required.
